compiler/one-cmds/how-to-use-one-commands.txt

```diff
@@ -0,0 +1,114 @@
+3) one-quantize will quantize models. Depending on the NPUs.
+4) one-codegen will compile to binary codes.
+
+
+one-import
+-----------
+
+one-import will invokes one-import-* commands.
+
+Syntax: one-import [framework] [options]
+
+Currently supported frameworks are 'tf', 'tflite' for TensorFlow and TensorFlow
+lite.
+
+
+one-import-tf
+-------------
+
+This will convert TensorFlow model (.pb) file to our circle model. You can also
+directly call this command. one-import-tf invokes tf2tfliteV2.py script that
+will internally use TensorFlow lite converter and then invoke tflite2circle
+converter to convert tflite model to circle model. 
+
+As tf2tfliteV2.py runs TensorFlow lite converter, you need to have TensorFlow
+installed in your system. We recommand to use 2.3.0rc0 for now.
+
+We provide python virtual environment and one-import-tf will enter and leave
+this environment so that you don't need to explictly 'activate' virtual
+environment.
+
+
+one-import-tflite
+-----------------
+
+You can use one-import-tf to convert TensorFlow lite model (.tflite) file to our
+circle model. Internally this will invoke tflite2circle.
+
+
+one-optimize
+------------
+
+one-optimize provides network or operator transformation shown below
+
+Current transformation options are
+- fuse_bcq: This enables Binary-Coded-bases Quantized DNNs
+   - read https://arxiv.org/abs/2005.09904 for detailed information
+- fuse_instnorm: This will convert subnetwork that can was generaged from
+  instance normalization operator to one InstanceNormalization operator that our
+  onert provides for faster execution.
+- resolve_customop_add: This will convert Custom(Add) to normal Add operator
+- resolve_customop_batchmatmul: This will convert Custom(BatchMatMul) to
+  normal BatchMatMul operator
+- resolve_customop_matmul: This will convert Custom(MatMul) to normal MatMul
+  operator
+
+
+one-quantize
+------------
+
+one-quantize will quantize float32 model to uint8 so that the model can benefit
+for speed that our onert runtime and NPU provides. For convolution type
+operators we currently support layer-wise quantization. Later we will support
+int16 and channel-wise quantization.
+
+Internally this calls circle-quantizer and record-minmax tools.
+
+
+one-pack
+--------
+
+one-pack will generate a package from circle model to nnpkg for our onert
+runtime.
+
+Output is a folder with the model(s) and meta information
+
+ex) if you have a model named '20200719.circle' and want to pack to 'testnnpack'
+
+$ one-pack -i 20200709.circle -o testnnpack
+
+$ tree testnnpack
+testnnpack
+└── 20200709
+    ├── 20200709.circle
+    └── metadata
+        └── MANIFEST
+
+
+one-codegen
+-----------
+
+one-codegen, like one-import will invoke backend code generation commands.
+As of now, our ONE repo does not provide any code generation commands yet.
+
+Syntax: one-codegen [framework] [options]
+
+This will invoke [framework]-compile command if available.
```

Comment:
Same suggestion.

```suggestion
This will invoke [target-backend]-compile command if available.
```

And with additions

```suggestion
This will invoke [target-backend]-compile command if available.

ex) If you have a model called '20200719.circle' and you want to generate code 
that runs on the cpu of the `armv7l` architecture, `20200709-armv7l.bin`. 
(Of course, we don't provide this `cpu-compile` yet.)

$ one-codegen cpu -arch armv7l -i 20200709.circle -o 20200709-armv7l.bin
```